config.toml

```diff
@@ -150,7 +150,10 @@ key_info_manager = "sqlite-manager"
 # (Required for this provider) Path to the location of the dynamic library loaded by this provider.
 # For the PKCS 11 provider, this library implements the PKCS 11 API on the target platform.
 #library_path = "/usr/local/lib/softhsm/libsofthsm2.so"
-# (Optional) PKCS 11 slot that will be used by Parsec.
+# (Optional) PKCS 11 serial number of the token that will be used by Parsec.
+# If the token serial number is entered, the slot has the token with the provided serial number will be used, and a warning should be issued in case of a mismatch
```

Comment:
Trying to make this a bit clearer:
```suggestion
# If the token serial number is entered, then the slot that has the provided serial number will be used. Otherwise, if both `serial_number` and `slot_number` are given but do not match, a warning is issued and serial number takes precedence.
```